include/mbedtls/psa_util.h

```diff
@@ -0,0 +1,162 @@
+/**
+ * \file psa_compat.h
+ *
+ * \brief Utility functions for the use of the PSA Crypto library.
+ *
+ * \warning This function is not part of the public API and may
```

Comment:
Minor: "this file" or "these functions".

Reply:
Fixing.

Reply:
Up.